controllers/endpoints_controller.js

```diff
@@ -0,0 +1,4 @@
+const endpoints = require("../endpoints.json")
+exports.getEndpoints = (req, res, next) => {
+res.status(200).send({message: Object.keys(endpoints)});
```

Comment:
When you return the endpoints, you'd want them to be returned as a JSON Object. 
This will allow for better readability when you come back to reference your endpoints in the frontend part of the course.